homeassistant/components/services.yaml

```diff
@@ -571,3 +571,32 @@ counter:
       entity_id:
         description: Entity id of the counter to reset.
         example: 'counter.count0'
+
+abode:
+  change_setting:
+    description: Change an Abode system setting.
+
+    fields:
+      setting:
+        description: Setting to change.
+        example: 'beeper_mute'
+        
+      value:
+        description: Value of the setting.
+        example: '1'
+
+  capture_image:
+    description: Request a new image capture from a camera device.
+    
+    fields:
+      entity_id:
+        description: Entity id of the camera to request an image.
+        example: 'camera.downstairs_motion_camera'
+        
+  trigger_quick_action:
+    description: Trigger an Abode quick action.
+    
+    fields:
+      entity_id:
+        description: Entity id of the quick action to trigger.
+        example: 'binary_sensor.home_quick_action'
```

Comment:
No blank line at end of file.

Reply:
Blank line was added -- file is now 603 lines long with the 603rd line blank.